### ISSUE TITLE
pyyaml version needs update

### ISSUE THREAD

**Issue by LindsayHill** [StackStorm-Exchange/stackstorm-jira#24]
The `pyyaml` version specified in requirements.txt has a security issue - see CVE-2017-18342

This needs to be updated. 

@sinnershiki is this something you could check with your system? It should be OK to just upgrade to `~> 4.2b1`, but it needs testing.

**Comment by sinnershiki** [comment]
I'll try to check it.

**Comment by sinnershiki** [comment]
Action OK.
```
root@90d1c60808e8:/# /opt/stackstorm/virtualenvs/jira/bin/pip install -U --pre pyyaml==4.2b4
root@90d1c60808e8:/# st2ctl restart
root@90d1c60808e8:/# /opt/stackstorm/virtualenvs/jira/bin/pip list | egrep "jira|cryptography|PyJWT|PyYAML"
DEPRECATION: The default format will switch to columns in the future. You can use --format=(legacy|columns) (or define a format=(legacy|columns) in your pip.conf under the [list] section) to disable this warning.
cryptography (2.4.1)
jira (2.0.0)
PyJWT (1.6.4)
PyYAML (4.2b4)
You are using pip version 9.0.1, however version 19.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
root@90d1c60808e8:/# st2 run jira.assign_issue assignee="hoge" issue="HOGE-XX"
.
id: 5c5bfe96148aa10886bcbed8
status: succeeded
root@90d1c60808e8:/# st2 run jira.comment_issue comment_text="pyyaml==4.2b4 test" issue_key="HOGE-XX"
..
id: 5c5bfebf148aa10886bcbedb
status: succeeded
```

**Comment by sinnershiki** [comment]
Sensor OK.
```
# /opt/stackstorm/st2/bin/st2sensorcontainer --config-file=/etc/st2/st2.conf --debug --sensor-ref=jira.JIRASensorForAPIv2
2019-02-07 09:50:44,524 DEBUG [-] 1 active sensor(s)
2019-02-07 09:50:49,526 DEBUG [-] 1 active sensor(s)
2019-02-07 09:50:54,538 DEBUG [-] 1 active sensor(s)
2019-02-07 09:50:59,542 DEBUG [-] 1 active sensor(s)
2019-02-07 09:51:04,544 DEBUG [-] 1 active sensor(s)
2019-02-07 09:51:09,513 DEBUG [-] 1 active sensor(s)
2019-02-07 09:51:12,139 DEBUG [-] Resetting dropped connection: jira.hogehoge.com
2019-02-07 09:51:13,015 DEBUG [-] Added trace_context None to trigger jira.issues_tracker_for_apiv2.
2019-02-07 09:51:13,028 DEBUG [-] Dispatching trigger jira.issues_tracker_for_apiv2 with payload
```

**Comment by LindsayHill** [comment]
Thanks for the tests. Resolved by #26
